package.json

```diff
@@ -33,28 +33,28 @@
   "devDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.2",
     "@openzeppelin/contracts": "^3.4.1",
-    "@typechain/ethers-v5": "^1.0.0",
-    "@types/chai": "^4.2.17",
-    "@types/chai-as-promised": "^7.1.3",
-    "@types/lodash": "^4.14.168",
+    "@typechain/ethers-v5": "^7.0.1",
+    "@types/chai": "^4.2.18",
+    "@types/chai-as-promised": "^7.1.4",
+    "@types/lodash": "^4.14.170",
     "@types/mocha": "^8.2.2",
-    "@types/node": "^14.14.43",
+    "@types/node": "^14.17.3",
     "chai": "^4.3.4",
     "chai-as-promised": "^7.1.1",
-    "hardhat": "^2.2.1",
+    "hardhat": "^2.3.2",
     "minimist": "^1.2.5",
     "prettier": "^1.19.1",
-    "prettier-plugin-solidity": "1.0.0-beta.10",
-    "solhint": "^3.3.4",
-    "ts-node": "^9.1.1",
-    "typechain": "^2.0.1",
-    "typescript": "^4.2.4"
+    "prettier-plugin-solidity": "1.0.0-beta.11",
```

Comment:
We will not be able to go higher on this version until https://github.com/thehubbleproject/hubble-contracts/issues/568